pyro/infer/trace_tail_adaptive_elbo.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import absolute_import, division, print_function
+
+import warnings
+
+import torch
+
+from pyro.infer.trace_elbo import Trace_ELBO
+from pyro.infer.util import is_validation_enabled, check_fully_reparametrized
+
+
+class TraceTailAdaptive_ELBO(Trace_ELBO):
+    """
+    Interface for Stochastic Variational Inference with an adaptive
```

Comment:
nit: it would be nice to split this into three paragraphs: summary, note on nonreparam, note on loss computation.